sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 - `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
 
+### New Features
+
+- Add `RequestFailException` deriving from `std::runtime_error`.
```

Comment:
`RequestFailedException`, I believe it's typo.

Reply:
Good catch. We decided to rename this to `HttpRequestException` and make some other changes, so let's update the CL with that change.